apps/o2-blocks/src/github-issue-template/editor.js

```diff
@@ -73,7 +68,7 @@ const Edit = ( {
 
 const View = ( { title, userOrOrg, repo, ...rest } ) => (
 	<Shell
-		title={ `${ __( 'Create Issue:' ) } ${ title }` }
+		title={ __( 'Create Issue' ) + ( title ? ` "${ title }"` : '' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.